packages/dd-trace/src/config.js

```diff
@@ -478,9 +398,13 @@ class Config {
 
     const defaults = setHiddenProperty(this, '_defaults', {})
 
+    this._setValue(defaults, 'appsec.apiSecurity.enabled', true)
+    this._setValue(defaults, 'appsec.apiSecurity.requestSampling', 0.1)
+    this._setValue(defaults, 'appsec.blockedTemplateGraphql', undefined)
     this._setValue(defaults, 'appsec.blockedTemplateHtml', undefined)
     this._setValue(defaults, 'appsec.blockedTemplateJson', undefined)
     this._setValue(defaults, 'appsec.enabled', undefined)
+    this._setValue(defaults, 'appsec.eventTracking', { enabled: true, mode: 'safe' })
```

Comment:
shouldn't that be split in different props instead ???
```suggestion
    this._setValue(defaults, 'appsec.eventTracking.enabled', true)
    this._setValue(defaults, 'appsec.eventTracking.mode', 'safe')
```
and also down below ?

Reply:
there is no [mapping](https://github.com/DataDog/dd-go/blob/prod/trace/apps/tracer-telemetry-intake/telemetry-payload/static/config_norm_rules.json) for appsec.eventTracking.enabled or appsec.eventTracking.mode. This means when other languages send this configuration through telemetry to the backend, they send appsec.eventTracking directly.

Reply:
Im going to ask the other teams to see if we should add the mapping maybe

Reply:
Hi Ida,
`enabled` field of `appsec.eventTracking` is just for internal use, it is inferred from `mode` [here](https://github.com/DataDog/dd-trace-js/blob/367c693c2e87a6314bfa139785a28e08c1f52342/packages/dd-trace/src/config.js#L369)

I think that we should send only `appsec.eventTracking.mode` and add this key in mappings. It is the reason why the other libraries are sending only one field and not two.

Reply:
thank you @uurien I'll make the changes

Reply:
the changes have been made :)

Reply:
@ida613 did you also change the mapping in the backend ?

Reply:
yup

Reply:
https://github.com/DataDog/dd-go/pull/144559/files

Reply:
I merged the above PR, approving this one :+1: